### DISCUSSION TITLE
TypeScript icon please

### DISCUSSION THREAD

**Discussion by eaworks** [elrumo/macOS_Big_Sur_icons_replacements#1471] [Icon request]
file and application icon. https://www.typescriptlang.org/branding

**Comment by itjustcrashed** [comment]
It's waiting approval, but here is the PNG:
![TypeScript](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/151413500/19bb35b0-c71e-40d6-b32b-fc801b1d4318)

**Reply by eaworks** [reply]
tanks :)

**Reply by itjustcrashed** [reply]
wewcome
